tests/test_loss_weights.py

```diff
@@ -0,0 +1,19 @@
+"""Test the PyKEEN sample weighters."""
+
+import unittest_templates
+
+import pykeen.triples.weights
+from tests import cases
+
+
+class RelationFrequencyLossWeighterTests(cases.LossWeightTestCase):
```

Comment:
heads up @mberr I added stubs for tests

Reply:
filled in https://github.com/pykeen/pykeen/pull/1534/commits/8c7e389d051c5bc25a9d8b75ed9fbf07251062d5